src/parse-args.ts

```diff
@@ -89,6 +91,9 @@ export default function(input: string[]): Args {
       case 'update':
         flags.update = true
         break
+      case 'unsafe':
+        flags.unsafe = true
```

Comment:
you could simplify ENV handling by moving it here; i don't know if we do that otherwise (as with, say, verbosity or debug).

Reply:
worth starting a discussion about, definitely. experience teaches us that there are sometimes unwritten design principles at play; though, in general, providing for either flag or env control allows a wider variety of use-cases.

Reply:
What about `PKGX_UNSAFE_INSTAL=0`? It means false, so we shouldn't consider it to be set. And if it's this, we should check for the other.

Reply:
I would think if either --unsafe or PKGX_UNSAFE_INSTALL != 0, then we do an unsafe install. PKGX_UNSAFE_INSTALL=0 should basically do nothing.

Reply:
You mean, the env should take priority?

Then, it needs to be like:
```ts
function is_unsafe(unsafe: boolean): boolean {
    const env = Deno.env.get("PKGX_UNSAFE_INSTALL");
    if (env) {
        // If present, it takes priority
        return parseInt(env) ? true : false;
    } else {
       // Otherwise, check flag
       return unsafe;
    }
```